exercises/supervised-classification/ex_rnw/sol_lda_qda.Rnw

```diff
@@ -1 +1,80 @@
-foo
\ No newline at end of file
+\begin{enumerate}[a)]
+
+  \item \begin{enumerate}[i)]
+  
+    \item As the data seem to be pretty symmetric conditional on the 
+    respective class, we estimate the class means to lie roughly in the middle 
+    of the data clusters: $\hat \mu_1 = 1$, $\hat \mu_2 = 7$, $\hat \mu_3 = 4$.
+    \textcolor{blue}{@Ludwig: die Datenlage ist tatsächlich etwas suboptimal, 
+    weil die aus einer Unif gezogen sind statt aus einer NV - genau genommen 
+    passen halt QDA und LDA hier nicht gut}
```

Comment:
Ja, das stimmt. Grund war, dass ich nicht wollte, dass die Datenpunkte an den Intervallgrenzen so "ausfransen" für die Klausur. Ich würde es auch so lassen, dafür aber in der a)iv) einen Hinweis dazu geben?